src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -1339,22 +1317,29 @@ export default class AircraftModel {
      */
     scoreWind(action) {
         const score = 0;
-        const components = this.getWind();
         const isWarning = true;
 
+        let wind;
+
+        if (this.isArrival()) {
+            wind = this.fms.arrivalAirportModel.getWindForRunway(this.fms.arrivalRunwayModel);
```

Comment:
I think we should calculate the wind relative to the aircraft instead of the runway.

Reply:
how do you mean?  currently wind is very tied to runways, I'm curious to hear what you have in mind there.

Reply:
Currently, we are using the runway heading to calculate the crosswind (i.e. runway 18 = 180 degree). Instead of using the runway heading, we could simply use the aircraft's current heading. During takeoff and landing, the aircraft's heading should be equal to the runway heading.

Reply:
now works based on aircraft heading